app/views/components/_notice.html.erb

```diff
@@ -3,8 +3,15 @@
     description_text ||= ""
     description_govspeak ||= ""
   %>
+
   <section class="app-c-notice" aria-label="Notice">
-    <h2 class="app-c-notice__title"><%= title %></h2>
+
```

Comment:
A bit of inconsistent line spacing in this file - line gaps 6 and 8 seem unnecessary, but there probably should be one between 16 and 18.